### PR TITLE
Update case_report and panel_reports to use bootstrap 5

### PR DESCRIPTION
**Pull request by TereseBo**
This PR replaces bootstrap 4 in the report_base with bootstrap 5.1.3. This results in minor visual changes due to differences between the bootstrap versions such as changes in the hues of colors and the visual expression of borders. In addition to replacing redundant classes inline styling have been added to retain the previous look of the reports.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by TB

### CODE REVIEW

**Comment by TereseBo** [comment]
A case_report on main:
image

image

For a different case:
image

image


Report with comments:
image

**Comment by TereseBo** [comment]
On this branch:
The downloaded report for STR and SMN panel hits for a case:
image

Downloaded pdf panel export:
[391_1.2_2024-04-04_scout.pdf](https://github.com/Clinical-Genomics/scout/files/14871797/391_1.2_2024-04-04_scout.pdf)

File: scout/server/templates/report_base.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 
 	{% block css %}
-	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
```
Comment (dnil) [inline scout/server/templates/report_base.html:7]: Let's add an integrity check for this one as well!

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 2]
```diff
@@ -12,44 +12,46 @@
...
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <a class="nav-link link-primary mx-2 " style="text-decoration: none !important;" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" rel="noopener" target="_blank">Case {{case.display_name}}</a>
 
-  <!-- Collect the nav links, forms, and other content for toggling -->
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-  </button>
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:20]: Not your doing, but this isn't really active right now anyway, is it? We may as well drop it. 😊

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 3]
```diff
@@ -12,44 +12,46 @@
 
 {% block body %}
 <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light">
-  <!-- Brand and toggle get grouped for better mobile display -->
-  <a class="nav-link" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" target="_blank">Case {{case.display_name}}</a>
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <a class="nav-link link-primary mx-2 " style="text-decoration: none !important;" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" rel="noopener" target="_blank">Case {{case.display_name}}</a>
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:17]: We don't really need the opener link so all good to be safe, but `noopener` won't do much here as the source and dest are the same site. 😊

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 4]
```diff
@@ -153,35 +155,35 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
...
+	            <span style="color:green">{{case.status.upper()}}</span>
 	          {% else %}
 	            {{case.status.upper()}}
 	          {% endif %}
 	          </strong><br><br>
 	        </td>
 	      </tr>
       </table>
-      <table
+      <table class="table table-sm" aria-label="Case synopsis and phenotype information">
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:168]: 🥇

**Review by dnil — APPROVED** [review]
Nicely done - thank you! Ideally we would merge the bug fixes to the issues you found with the patch release lined up, and then this to main for the next feature release. 
I'm starting to think we might just skip the patch release, but maybe wait a little with merging so we keep a quick option to release without needing a hot-fix branch.

File: scout/server/blueprints/cases/templates/cases/case_report.html (hunk 2 above)
Comment (TereseBo) [reply scout/server/blueprints/cases/templates/cases/case_report.html:20]: I think it's the button used to expand the menu when having a small screen/window?

image

Comment (TereseBo) [reply scout/server/blueprints/cases/templates/cases/case_report.html:20]: It's the button to expand the menu if using a small screen/window I believe?

image

Comment (TereseBo) [reply scout/server/blueprints/cases/templates/cases/case_report.html:20]: image

**Pull request merged**